### PR TITLE
feat(live-11604): clean up swap legacy code and flags

### PR DESCRIPTION
**Pull request by liviuciulinaru**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

- removed unsued swap legacy code
- removed old swap feature flags
- always load swap live app 
- ptxSwapDemoThree flag is used only for pointing the manifest id of live app

https://github.com/user-attachments/assets/1db1344f-5a5e-483f-8d45-a6f0fdea4a58

### ❓ Context

- **JIRA or GitHub link**: 
https://ledgerhq.atlassian.net/browse/LIVE-11603
https://ledgerhq.atlassian.net/browse/LIVE-11604
https://ledgerhq.atlassian.net/browse/LIVE-11605
https://ledgerhq.atlassian.net/browse/LIVE-11606
https://ledgerhq.atlassian.net/browse/LIVE-14096

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: libs/ledger-live-common/src/exchange/swap/hooks/v5/useFilteredProviders.ts [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 import { fetchAndMergeProviderData } from "../../../providers/swap";
-import { getEnv } from "@ledgerhq/live-env";
 
 export const useFilteredProviders = () => {
   const [providers, setProviders] = useState<string[]>([]);
   const [loading, setLoading] = useState(true);
   const [error, setError] = useState<unknown>(null);
 
-  const ptxSwapMoonpayProviderFlag = useFeature("ptxSwapMoonpayProvider");
   const ptxSwapExodusProviderFlag = useFeature("ptxSwapExodusProvider");
```
Comment (CremaFR) [inline libs/ledger-live-common/src/exchange/swap/hooks/v5/useFilteredProviders.ts:11]: Why only remove MoonPay and not Exodus

File: libs/ledger-live-common/src/featureFlags/defaultFeatures.ts [hunk 2]
```diff
@@ -399,36 +398,13 @@ export const DEFAULT_FEATURES: Features = {
...
-  },
-
-  ptxSwapLiveAppDemoOne: {
-    enabled: false,
-    params: {
-      manifest_id: "swap-live-app-demo-1",
-    },
-  },
-
   ptxSwapLiveAppDemoThree: {
```
Comment (CremaFR) [inline libs/ledger-live-common/src/featureFlags/defaultFeatures.ts:401]: We should remove the demo3 in the name and only use ptxSwapLiveApp

File: apps/ledger-live-desktop/src/renderer/modals/Platform/Exchange/CompleteExchange/Body.tsx [hunk 3]
```diff
@@ -195,7 +195,7 @@ const Body = ({ data, onClose }: { data: Data; onClose?: () => void | undefined
   const handleSellTransaction = (operation: Operation, result: ResultsState) => {
     handleTransactionResult(result, operation);
   };
-  const isDemo3Enabled = useIsSwapLiveFlagEnabled("ptxSwapLiveAppDemoThree");
+  const isDemo3Enabled = true;
```
Comment (CremaFR) [inline apps/ledger-live-desktop/src/renderer/modals/Platform/Exchange/CompleteExchange/Body.tsx:198]: IsSwapLiveAppEnabled would be a better name here

Comment (kallen-ledger) [reply apps/ledger-live-desktop/src/renderer/modals/Platform/Exchange/CompleteExchange/Body.tsx:198]: If this is a hardcoded const do we need it?

Comment (liviuciulinaru) [reply apps/ledger-live-desktop/src/renderer/modals/Platform/Exchange/CompleteExchange/Body.tsx:198]: removed

File: libs/ledger-live-common/src/featureFlags/defaultFeatures.ts (hunk 2 above)
Comment (liviuciulinaru) [reply libs/ledger-live-common/src/featureFlags/defaultFeatures.ts:401]: replaced

**Review by CremaFR — APPROVED** [review]
🔥🥵🥰✨💪🚀💖🍷

File: .changeset/tricky-dryers-check.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+"@ledgerhq/types-live": patch
```
Comment (themooneer) [inline .changeset/tricky-dryers-check.md:2]: can you please replace all `patch` to `minor`

**Review by themooneer — APPROVED** [review]
LGTM

**Review by Couto — APPROVED** [review]

**Review by kallen-ledger — APPROVED** [review]
Nice 🔥

**Pull request merged**